src/seedu/addressbook/formatter/Formatter.java

```diff
@@ -0,0 +1,62 @@
+package seedu.addressbook.formatter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.ui.TextUi;
+
+/**
+ * Formatter for the output of TextUi
+ */
+public class Formatter {
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+    private final TextUi textUi;
+
+    public Formatter(TextUi textUi) {
+        this.textUi = textUi;
+    }
+
+    /**
+     * Shows a list of persons to the user, formatted as an indexed list.
+     * Private contact details are hidden.
+     */
+    public void showPersonListView(List<? extends ReadOnlyPerson> persons) {
+        final List<String> formattedPersons = new ArrayList<String>();
+        for (ReadOnlyPerson person : persons) {
+            formattedPersons.add(person.getAsTextHidePrivate());
+        }
+        showToUserAsIndexedList(formattedPersons);
+    }
+
+    /**
+     * Shows a list of strings to the user, formatted as an indexed list.
+     */
+    public void showToUserAsIndexedList(List<String> list) {
```

Comment:
This kinda increases the coupling (Heads up! coupling will be covered later and you will realize what I mean!) 
Also it violates SRP, think how?
A better approach is to do only formatting related stuff here and return the formatted string to the caller.